queries.sql

```diff
@@ -135,3 +135,81 @@ SELECT full_name, COUNT(full_name) AS amount FROM owners
 JOIN animals ON animals.owner_id = owners.id
 GROUP BY full_name
 ORDER BY amount DESC LIMIT 1;
+
+
+
+-- ---------------- --
+-- FORTH MILESTONE --
+-- ---------------- --
+
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name animal, vt.name vet, vs.visit_date FROM animals a
+JOIN visits vs ON a.id = vs.animal_id
+JOIN vets vt ON vt.id = vs.vet_id
+WHERE vt.name = 'William Tatcher'
+ORDER BY vs.visit_date DESC LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT ve.name, COUNT(DISTINCT v.animal_id) AS distinct_animals
+FROM visits v
+JOIN vets ve ON v.vet_id = ve.id
+WHERE ve.name = 'Stephanie Mendez'
+GROUP BY ve.name;
```

Comment:
- [ ] Your screenshot that answers this question does not show the query result which is important. Kindly take another screenshot that shows the result.

Reply:
Hello @bobb-Rob. Thanks for reviewing the PR.
I thank you haven't checked the screenshots clearly. 
It exists on this PR.

![image](https://github.com/M-Anwar-Hussaini/Vet-Clinic/assets/131291114/927fd998-0eae-48a3-be50-8fe9705876ce)